plugins/modules/network_settings_intent.py

```diff
@@ -1949,6 +1978,104 @@ def get_diff_deleted(self, config):
 
         return self
 
+    def verify_diff_merged(self, config):
+        """
+        Validating the DNAC configuration with the playbook details
+        when state is merged (Create/Update).
+
+        Parameters:
+            config (dict) - Playbook details containing Global Pool,
+            Reserved Pool, and Network Management configuration.
+
+        Returns:
+            self
+        """
+
+        self.get_have(config)
+        self.log(str(self.have))
+        self.log(str(self.want))
+        if config.get("global_pool_details") is not None:
+            obj_params = self.get_obj_params("GlobalPool")
+            self.log(str(self.want.get("wantGlobal")))
```

Comment:
How do we know that we are actually printing wantGlobal? 
Can we enhance our log API something like our print API

```
self.log(str(self.want.get("wantGlobal")), start="want Global", sep = ":")

    def log(self, message, frameIncrement=0, start="", end="", sep =""):
        """Log messages into dnac.log file"""

        if self.dnac_log:
            message = "Module: " + self.__class__.__name__ + ", " + start + sep + message + end
            log(message, (1 + frameIncrement))

```

You can take it as another work item and do it..